tensorflow/lite/micro/kernels/kernel_util.h

```diff
@@ -18,6 +18,8 @@ limitations under the License.
 
 #include <cstdint>
 
+#define FLATBUFFERS_LOCALE_INDEPENDENT 0
```

Comment:
It seems a bit odd to be including flexbuffer utility functions in this file, and to have to define a macro in the code before we include the header. We should probably move these functions to their own .h and .cc files. Is there a reason that TFL mobile doesn't already require these?

Reply:
Sorry we did not use any  flexbuffer utility functions,so we will revert this change